cypress/e2e/home/home-page-eu.cy.ts

```diff
@@ -0,0 +1,29 @@
+describe('EU Language pop up in the navigation', () => {
+    //before every check visit base url
+    beforeEach(() => {
+        cy.visit(`${Cypress.env('base_url_eu') || 'https://eu.deriv.com'}`)
+    })
+    it('language popup opens perfectly in eu', () => {
+        cy.get('#radix-4').click()
+    })
+
+    it('EU Language popup Should contain all language items in the dropdown', () => {
+        // Visit the webpage or navigate to the relevant page
+
+        cy.get('#radix-4').click()
```

Comment:
Can have the parent tag to make it more readable
```suggestion
        cy.get('header #radix-4').click()
```